pom.xml

```diff
@@ -71,6 +71,11 @@
             <artifactId>commons-io</artifactId>
             <version>${commons-io.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>30.0-jre</version>
```

Comment:
can this version number please be moved into the properties section where other version are managed